config/jobs/periodic/knative/operator/main/operator-main.gen.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+  - name: knative-operator-main-periodic
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    cron: "20 0 * * *"
+    extra_refs:
+      - base_ref: main
+        org: valen-mascarenhas14
```

Comment:
Have forgotten to change the org to `ppc64le-cloud` ?

Reply:
changed the org